src/bin/travis/testSbt.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Docs may be inconsistent, but this is the one I was working off:

> When using custom scripts they should be executable (for example, using chmod +x) and contain a valid shebang line such as /usr/bin/env sh, /usr/bin/env ruby, or /usr/bin/env python.

https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Installation-Step